dev/patching-deps.nix

```diff
@@ -0,0 +1,18 @@
+final: prev: {
+  arpeggio =
+    (prev.arpeggio.override {
+      # We cannot patch a wheel, we must build from source to apply a patch.
```

Comment:
While you're right about not being able to patch a wheel, it _is_ possible to do fixups in `postInstall`/`postFixup`.
Possibly worth mentioning somewhere?

Reply:
Done! What's a fixup someone would do on a wheel? `patchelf`? Please take a look at my update.

Reply:
I meant that you can still patch sources, they just have to be patched after the install into the Nix store has happened.
`patchelf` isn't particularly interesting, as `autoPatchelfHook` is already applied to wheels.

Reply:
Ahhh of course! Just updated.